featureflags_client/http/managers/aiohttp.py

```diff
@@ -30,6 +30,7 @@ def __init__(  # noqa: PLR0913
         project: str,
         variables: List[Variable],
         defaults: Union[EnumMeta, Type, Dict[str, bool]],
+        values_defaults: EnumMeta | Type | dict[str, int | str] | None = None,
```

Comment:
Since we are supporting >=3.9 version of python, I suggest to use Union annotation